assertions.js

```diff
@@ -0,0 +1,116 @@
+/*!
+ * Copyright (c) 2024 Digital Bazaar, Inc.
+ */
+import chai from 'chai';
+
+const should = chai.should();
+// RegExp with bs58 characters in it
+const bs58 =
+  /^[123456789ABCDEFGHJKLMNPQRSTUVWXYZabcdefghijkmnopqrstuvwxyz]+$/;
+
+/**
+ *  Regex to test conformity to base-64-url-no-pad character set.
+ *
+ * @see {@link https://www.w3.org/TR/vc-data-integrity/#multibase-0} for
+ * specification.
+*/
+const BASE_64URL_NOPAD_REGEX = /^[A-Za-z0-9\-_]+$/;
```

Comment:
We should pick a naming style for constants like these and stick with it. These two are very far apart "linguistically."

Ideally, the `bs58` one should be a bit more verbose (i.e. `base58`), and the base64 one should use some sane "spacing" (i.e. `base64url` rather than `base_64url`). Essentially, match the search terms other devs will use when hunting around the code.

Reply:
totally agree here and camelCase is how we generally handle things, but that is breaking as other test suites depend on the existing naming.

Reply:
issue made here: https://github.com/w3c-ccg/data-integrity-test-suite-assertion/issues/55 feel free to expand the comments.